File: 1561-maximum-number-of-coins-you-can-get/1561-maximum-number-of-coins-you-can-get.py
class Solution:
    def maxCoins(self, piles: List[int]) -> int:
        res = 0
        piles.sort(key=lambda x: x, reverse=True)
        for i in range(1, len(piles) * 2 // 3, 2):
            res += piles[i]
        return res

File: 1561-maximum-number-of-coins-you-can-get/1561-maximum-number-of-coins-you-can-get.js
/**
 * @param {number[]} piles
 * @return {number}
 */
var maxCoins = function(piles) {
    piles.sort((a, b) => b - a)
    let res = 0
    while (piles.length) {
        piles.shift()
        res += piles.shift()
        piles.pop()
    }
    return res
};